apps/experiments/urls.py

```diff
@@ -43,7 +43,7 @@
     path("table/", views.ExperimentTableView.as_view(), name="table"),
     path("e/<int:experiment_id>/", views.single_experiment_home, name="single_experiment_home"),
     path("e/<int:experiment_id>/sessions-table/", views.ExperimentSessionsTableView.as_view(), name="sessions-list"),
-    path("e/<int:pk>/set-default/", views.set_default_experiment, name="set_default_experiment"),
+    path("e/<int:pk>/", views.set_default_experiment, name="set-default-experiment"),
```

Comment:
just looking at the urls below there are a few that follow the patter `e/<int:pk>/{action}/` which makes me thing this should follow the same pattern. Also, a POST request to `/a/team/e/56/` would typically be an update request so this would break that pattern which makes it a possible source of confusion.